### ISSUE TITLE
The problem of the M in the paper

### ISSUE THREAD

**Issue by ckqsars** [ifzhang/FairMOT#97]
It is a nice work.  Thank you for your share.
I have some problems about the M in your heat map.
You  use the sum of N Gaussian function  object center point to represent the M value. However when the xy is same as the x^i_c and y^i_c the sum of M_xy is bigger than 1, because sum others.
In your loss of heatmap the positive is M=1, other is negative.
whether use max replace sum is more reasonable? 
could you explain when the heatmap is same as groundtruth, but the value of M is not 1?

**Comment by ifzhang** [comment]
Yes, max is more reasonable. When using sum, we need to add a clip option to keep the value between 0 and 1. In fact, the code is using max instead of sum:
https://github.com/ifzhang/FairMOT/blob/1851158a1bc025da7e6cb839ddef9d14e33b404a/src/lib/utils/image.py#L140

**Comment by ret-1** [comment]
@ckqsars In fact, in CenterNet on which the detection branch is built upon, the authors said that max is used
<img width="573" alt="截屏2022-03-08 下午4 14 04" src="https://user-images.githubusercontent.com/45256257/157194755-9596eec7-38e8-4d4b-b7ce-bb207092ee94.png">
